docker/meltano/Dockerfile

```
@@ -0,0 +1,27 @@
+ARG PYTHON_VERSION
+
+FROM python:${PYTHON_VERSION}-slim as build
```

Nit: what do you think of dropping the `as build` here? Whenever I see a name here I expect a multi-build process, which we don't have here.

```suggestion
FROM python:${PYTHON_VERSION}-slim
```

If there's an advantage of having this clause added, I don't mind keeping it.

---

Good catch. That's a holdover from when I was doing this with a multi-stage build. I'll remove it.